### ISSUE TITLE
A11y_AzureDataStudio_AzureSettings_ScreenReader:VoiceOver does not announce the order list for the location combo box .

### ISSUE THREAD

**Issue by V-sheman** [microsoft/azuredatastudio#7938]
GitHubTags: #SQL Azure Data Studio;#A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#WCAG1.3.1;#DesktopApp;#Voice Over;#Mac;
Environment Details:
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
Additional Details:
Voice over
Steps to Reproduce :
1.Install /Launch the application Azure data studio.
2.Navigate to connections and click on more action button.
3.Select “Deploy SQL Server big data cluster” from more action drop down.
4.Click on “Deploy SQL Server big data cluster”.
5.Check the check box I accept in the “select the deployment option” screen and click on select button.
6.Select any block and click on next button.
7.Open Voce over and observe the screen reading.
**Actual:**
Voice over does not announce the order list for the Location drop down list.
**Expected:**
Voice over should announce the order list for the Location drop down list .
**User impact**
Screen reader User will not able to know the total number of the list drop down.
**Attachment for reference**
[7938_A11y_AzureDataStudio_AzureSettings_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3760740/7938_A11y_AzureDataStudio_AzureSettings_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by alanrenmsft** [comment]
similar to #5971 seems to be an issue with VoiceOver
